services/checks/checks.go

```diff
@@ -408,8 +430,7 @@ func (s *Service) processResults(ctx context.Context, check check.Check, target
 	alertsIDs := make([]string, len(results))
 	for i, result := range results {
 		id := alertsPrefix + hashID(target.serviceID+"/"+result.Summary)
-		alert := makeAlert(id, check.Name, target, &result)
-		s.alertsRegistry.Add(id, 0, alert)
+		s.createAlert(id, check.Name, target, &result)
```

Comment:
**[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
G601: Implicit memory aliasing in for loop. (gosec)